Subject: [SERIAL] support for another Rockwell PNP modem
Date:	Mon, 03 Jan 2005 22:48:10 +0000
X-BK-ChangeSetKey: mzzhgg@de.rmk.(none)|ChangeSet|20050103224810|03818
Patch-mainline: yes
From: olh@suse.de


ChangeSet 1.2136.2.1, 2005/01/03 22:48:10+00:00, mzzhgg@de.rmk.(none)

	[SERIAL] support for another Rockwell PNP modem
	
	From: Lennart Poettering
	
	Trivial patch against Linux 2.6.10 which adds PNP support for a
	certain kind of Rockwell 28.8/ISA modems.



 8250_pnp.c |    2 ++
 1 files changed, 2 insertions(+)


diff -Nru a/drivers/serial/8250_pnp.c b/drivers/serial/8250_pnp.c
--- a/drivers/serial/8250_pnp.c	2005-01-03 22:08:07 -08:00
+++ b/drivers/serial/8250_pnp.c	2005-01-03 22:08:07 -08:00
@@ -250,6 +250,8 @@
 	/* Kortex International */
 	/* KORTEX 14400 Externe PnP */
 	{	"ROK0100",		0	},
+	/* Rockwell 28.8 */
+	{	"ROK4120",		0	},
 	/* Viking Components, Inc */
 	/* Viking 28.8 INTERNAL Fax+Data+Voice PnP */
 	{	"ROK4920",		0	},
-
To unsubscribe from this list: send the line "unsubscribe bk-commits-head" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

